package.json

```
@@ -103,7 +103,7 @@
     "lint-fix": "eslint --fix --ext .ts --ext .tsx src e2e packages/codemirror-lsp-client/src rust/kcl-language-server/client/src",
     "lint": "eslint --max-warnings 0 --ext .ts --ext .tsx src e2e packages/codemirror-lsp-client/src rust/kcl-language-server/client/src",
     "circular-deps": "dpdm --no-warning --no-tree -T --skip-dynamic-imports=circular src/index.tsx",
-    "diff-circular-deps": "yarn circular-deps | sed '$d' | grep -v '^yarn run' | tr -d '\n' > /tmp/circular-deps.txt && diff -w /tmp/circular-deps.txt ./known-circular.txt",
+    "diff-circular-deps": "./scripts/diff-circular-deps.sh",
```

Just a naming suggestion, since this seems related to the command above it. Feel free to ignore!

```suggestion
    "circular-deps:diff": "./scripts/diff-circular-deps.sh",
```